recipes/nest/meta.yaml

```
@@ -0,0 +1,96 @@
+{% set name = "nest-simulator" %}
+{% set version = "2.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nest/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: abfeb61719dec54da9477be035bef1d9d764f4e7663f63f6a6d9211f967e0490
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
```

Is this really `3.6` only or could it be built for both Python 3.6 and 3.7?
```suggestion
  skip: true  # [osx]
  skip: true  # [py<36]
```
If it has to be python 3.6 then use `[py!=36]` instead.

---

No, I think other python3 versioin should be ok.